lib/graphql/query/serial_execution/field_resolution.rb

```diff
@@ -24,18 +24,17 @@ def result
         # After getting the value from the field's resolve method,
         # continue by "finishing" the value, eg. executing sub-fields or coercing values
         def get_finished_value(raw_value)
+          execution_context.query.context.irep_node = irep_node
```

Comment:
Yeah, the other thing that gets me is the extra, possibly-needless assignment on a hot path. It's not cleaner code but maybe it was better the old way for that reason :S